multicast/__main__.py

```
@@ -256,7 +264,11 @@
 			tuple: A "tuple" set to None.
 
 		"""
-		return self.NoOp(*args, **kwargs)
+		_None_from_NoOp = self.NoOp(*args, **kwargs)
+		return (
+			True if _None_from_NoOp is None else False,
```

Ruff

Remove unnecessary `True if ... else False`

Remove unnecessary `True if ... else False`

(SIM210)